modules/http/src/main/java/org/glassfish/grizzly/http/HttpServerFilter.java

```diff
@@ -1195,6 +1204,24 @@ private void sendBadRequestResponse(final FilterChainContext ctx,
         commitAndCloseAsError(ctx, response);
     }
 
+    private void sendEntityTooLargeResponse(final FilterChainContext ctx,
+                                        final HttpResponsePacket response) {
+        if (response.getHttpStatus().getStatusCode() < 400) {
+            // 413 - Request Entity Too Large
+            HttpStatus.REQUEST_ENTITY_TOO_LARGE_413.setValues(response);
+        }
+        commitAndCloseAsError(ctx, response);
+    }
+
+    private void sendRequestUriTooLongResponse(final FilterChainContext ctx,
+                                               final HttpResponsePacket response) {
+        if (response.getHttpStatus().getStatusCode() < 400) {
+            // 414 - Request URI Too Long
+            HttpStatus.REQUEST_URI_TOO_LONG_414.setValues(response);
+        }
+        commitAndCloseAsError(ctx, response);
```

Comment:
Isn't this commiting and closing as error twice in the method and after the verification?